concepts/language-model.md

```diff
@@ -0,0 +1,81 @@
+---
+description: Language model
+---
+
+# Language model
+
+A language model takes text input (usually the already written text, called history) and outputs the next word or character.
```

Comment:
https://machinetranslate.org/style#minimal-content

> Avoid parentheses, except for defining common abbreviations.

Just "unpack" into a standalone sentence

> The input is often called a “context” or “history”, because it represents what has been written so far.